addons/website/views/website_templates.xml

```diff
@@ -1435,6 +1435,13 @@
     </xpath>
 </template>
 
+<!-- Area for blocks shared by all pages -->
+<template id="shared_blocks" inherit_id="website.layout" name="Shared blocks" active="True">
+    <xpath expr="//header" position="after">
```

Comment:
I would add this at the end of the `main`, seems more legit. If we ever wanted to use that for something else than popups, that's where I would want shared blocks. I'd like also to keep the header + main + footer being the 3 parts of the `#wrapwrap` element.